components/tesla_ble_vehicle/__init__.py

```
@@ -19,25 +19,21 @@
-        }
-    )
+    cv.Schema({
+        cv.GenerateID(CONF_ID): cv.declare_id(TeslaBLEVehicle),
+        # add support to set VIN (required)
+        cv.Required(CONF_VIN): cv.string,
+        cv.Optional(CONF_IS_ASLEEP): binary_sensor.binary_sensor_schema(icon="mdi:sleep").extend(),
+        cv.Optional(CONF_IS_UNLOCKED): binary_sensor.binary_sensor_schema(
+            device_class=binary_sensor.DEVICE_CLASS_LOCK,
+        ).extend(),
+        cv.Optional(CONF_IS_USER_PRESENT): binary_sensor.binary_sensor_schema(
+            icon="mdi:account-check", device_class=binary_sensor.DEVICE_CLASS_OCCUPANCY
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)